docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Deploy to two data centers with asynchronous replication
+headerTitle: Asynchronous Replication
+linkTitle: Asynchronous replication
+description: Enable deployment using asynchronous replication between two data centers.
+menu:
+  latest:
+    parent: create-deployments
+    identifier: async-replication-platform
+    weight: 633
+aliases:
+type: page
+isTocNested: true
+showAsideToc: false
+---
+
+You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+
+If you are using Yugabyte Platform to manage universes, you need to call the following REST API endpoint on your Yugabyte Platform instance for the producer universe and the consumer universe involved in the asynchronous replication between two data sources:
+
+```sh
+PUT /api/customers/<myUUID>/universes/<myUniUUID>/setup_universe_2dc
+```
+
+*myUUID* represents your customer UUID, and *myUniUUID* represents the UUID of the universe (producer or consumer). The request should include an `X-AUTH-YW-API-TOKEN` header with your Yugabyte Platform API key, as shown in the following example `curl` command:
```

Comment:
wouldn't it be easier if we spell these out like `customerUUID` and `universeUUID`?

Reply:
Fixed